src/components/FilterPanel/RadiusFilter.js

```diff
@@ -61,7 +61,11 @@ export default function RadiusFilter({ value, onChange }) {
 }
 
 export function isWithinRadius(item, zipCode, miles) {
-    if (isZipValid(zipCode)) {
+    if (
+        isZipValid(zipCode) &&
+        item.coordinates.latitude &&
```

Comment:
I looked into this (and it is `?.` btw).  For reference, this is [optional chaining notation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

Not really needed here because the object is guaranteed to have a `coordinates` key after going through `transformData` in `appointmentData.service.js`.  The problem is the source data doesn't have `entry.latitude` or `entry.longitude` -- thus the added test.

I was seeing the case in archived data set where `item.coordinates == { latitude: undefined, longitude: undefined }`